packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -130,6 +130,12 @@ export interface IButtonProps extends React.AllHTMLAttributes<HTMLAnchorElement
   onRenderMenu?: IRenderFunction<IContextualMenuProps>;
 
   /**
+   * Custom render function to show a divider between the
+   * primary button and the menu launcher for a split button.
```

Comment:
maybe another word than primary, since that is a button type.

Reply:
Also, is there reason to support rendering an entire JSX element vs just providing a divider style? Are there use cases where someone'll add more than just a div with bg/width etc? We could just take in an IStyle prop, and if it's provided, the divider will render with those styles. 

I partly bring this up because it means more unknown DOM in the middle of our button.

Reply:
I like that idea. I'll push it later tonight

Reply:
I like that idea. I'll push it later tonight

Reply:
I like that idea. I'll push it later tonight

Reply:
I'll probably just add it to the button styles.